packages/cozy-harvest/README.md

```diff
@@ -0,0 +1,10 @@
+Harvest
+=====
+
+Library to render a presentation for datas & configurations konnectors
```

Comment:
I think we can be more explicit, what do you think about:
> Cozy-Harvest will bring the logic, modules and components used in harvest application.
> This first version is just an "Hello World" version aimed to initialize the lib and make it available in NPM.

(My english is far to be perfect so maybe I made some typo.)

Reply:
Maybe one word about what is "harvest application" ?
Harvest application => generic presentation for collected data ?

Reply:
You're right, maybe :

> Cozy-Harvest will bring the logic, modules and components used in harvest application.
>
> Harvest applications will be apps which will allow to configure konnectors, setup authentification informations and check for synchronized data (And much more in the future).
> Each harvest application will be associated to only one konnector.
>
> This first version is just an "Hello World" version aimed to initialize the lib and make it available in NPM.